types/kavenegar/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "dom"],
```

```suggestion
        "lib": ["es6"],
```
why does this package need DOM being node module?

---

You're right. I added the DOM because I use `console.log` in testing file. But I removed it in my last commit.